core/trino-main/src/main/java/io/trino/operator/function/TableFunctionOperator.java

```diff
@@ -360,8 +362,8 @@ private static class HashStrategies
         final PagesHashStrategy prePartitionedStrategy;
         final PagesHashStrategy remainingPartitionStrategy;
         final PagesHashStrategy preSortedStrategy;
-        final List<Integer> remainingPartitionAndSortChannels;
-        final List<SortOrder> remainingSortOrders;
+        @Nullable
```

Comment:
Maybe make it an optional ?

Reply:
Yeah, makes sense in this operator. Done.